CHANGELOG.md

```diff
@@ -27,11 +27,17 @@ Swift 6.0
   `--experimental-swift-sdks-path` options on `swift build` are deprecated with replacements that don't have the
   `experimental` prefix.
 
+* [#7722]
+
+  For `swift sdk install` subcommand passed remote URLs as arguments, an additional `--checksum` option is now required, through which users of a Swift SDK can specify a checksum provided by
```

Comment:
A bit rephrased first bit: `An additional `--checksum` option  is now required for `swift sdk install` subcommand with remote URLs as arguments. `--checksum` allows users of a Swift SDK to specify a checksum provided by the SDK publisher.